core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/RequesterPaysSetupServiceSpec.scala

```diff
@@ -78,10 +79,9 @@ class RequesterPaysSetupServiceSpec extends AnyFlatSpec with Matchers with Mocki
     val initialBindings = Map(service.requesterPaysRole -> Set("serviceAccount:" + expectedEmail.client_email))
     service.googleServicesDAO.asInstanceOf[MockGoogleServicesDAO].policies.put(minimalTestData.workspace.googleProjectId, initialBindings)
 
-    // remove user from 1 workspace and check that it did not get removed from google bindings
     service.revokeUserFromWorkspace(userInfo.userEmail, minimalTestData.workspace).futureValue shouldBe List(expectedEmail)
     runAndWait(workspaceRequesterPaysQuery.userExistsInWorkspaceNamespace(minimalTestData.billingProject.projectName.value, userInfo.userEmail)) shouldBe true
-    service.googleServicesDAO.asInstanceOf[MockGoogleServicesDAO].policies.get(minimalTestData.workspace.googleProjectId) shouldBe Some(initialBindings)
+    service.googleServicesDAO.asInstanceOf[MockGoogleServicesDAO].policies.get(minimalTestData.workspace.googleProjectId) shouldBe Some(Map(service.requesterPaysRole -> Set.empty))
```

Comment:
Open Question: Should I remove the second workspace from this test? It seems kinda not relevant, but might reduce some coverage if we had a regression and it returned to working at the namespace level again?